packages/dx-react-grid/api/dx-react-grid.api.md

```diff
@@ -1558,9 +1558,9 @@ export class TableSummaryRowBase extends React_2.PureComponent<TableSummaryRowPr
     // (undocumented)
     static GROUP_ROW_TYPE: symbol;
     // (undocumented)
-    render(): JSX.Element;
+    render(): React_2.JSX.Element;
     // (undocumented)
-    renderContent(column: any, columnSummaries: any): JSX.Element;
+    renderContent(column: any, columnSummaries: any): React_2.JSX.Element;
```

Comment:
What happened here? Why docs have changed?

Reply:
Probably newer version of api lib fixed this. In dx-react-grid.api file there are many occurencies of React_2 prefix so it shouldn't affect the documentation page